src/main/java/core/basesyntax/strategy/handler/BalanceOperationHandler.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.CantPutFruitException;
+import java.util.Map;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public Integer processData(String fruitName, Integer quantity) {
+        if (getStorage().get(fruitName) != null) {
+            throw new CantPutFruitException(fruitName + "already exist in Storage");
+        }
+        return getStorage().put(fruitName, quantity);
+    }
+
+    private Map<String, Integer> getStorage() {
```

Comment:
It's a bit confusing why you need this method here, just call `Storage.getStorage()` when you need.

Reply:
> It's a bit confusing why you need this method here, just call `Storage.getStorage()` when you need.

I created this method here because I call Storage 2 times, so I decided to use a private method for this.

Reply:
It is not correct. Just use this Storage.getStorage()